LibNFS.xs

```diff
@@ -1178,6 +1205,14 @@ set (SV* self_sv, ...)
 
                 continue;
             }
+#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
```

Comment:
Should be one line below, but then `xsubpp` complains that "Code is not inside a function".

Reply:
Yeah I think inside an XSUB I’ve done plain C `if (NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE) { … }`, relying on the compiler to optimize away `if (1)` or `if (0)`.